can/simulator/FreeRTOSConfig.h

```diff
@@ -0,0 +1,175 @@
+/*
+ * FreeRTOS Kernel V10.0.1
+ * Copyright (C) 2017 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
```

Comment:
This file is just here to use the posix FreeRTOS port.

Reply:
wait this stuff all works on the posix port? a lot of it looks extremely cortex-m specific